daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -685,8 +685,8 @@ object Main extends Logging {
   }
 
   def createProcessorFromSchema(schema: URI, rootNS: Option[RefQName], path: Option[String],
-    tunables: Map[String, String],
-    mode: ValidationMode.Type): Option[DFDL.DataProcessor] = {
+                                tunables: Map[String, String],
```

Comment:
Your IDE made a bunch of whitespace changes I think. 
They make your change set look much more complex than it should be.